### PR TITLE
bumping UI and houston for 0.27 RC1

### PR DESCRIPTION
**Pull request by rdodev**
## Description

Bumps image for UI and houston for RC1 of 0.27

## 📋 Checklist

- [x] The PR title is informative to the user experience

### CODE REVIEW

File: charts/astronomer/values.yaml [hunk 1]
```diff
@@ -24,11 +24,11 @@ images:
     # httpSecret: ~
   houston:
     repository: quay.io/astronomer/ap-houston-api
-    tag: 0.26.9
+    tag: release-0.27
```
Comment (andriisoldatenko) [inline charts/astronomer/values.yaml:27]: 0.27.0?

**Review by andriisoldatenko — CHANGES_REQUESTED** [review]

File: charts/astronomer/values.yaml (hunk 1 above)
Comment (rdodev) [reply charts/astronomer/values.yaml:27]: As per CI logs `docker push astronomerinc/ap-astro-ui:release-0.27`

**Comment by vishwas-astro** [comment]
@rdodev - you need to change the platform version here as well: https://github.com/astronomer/astronomer/blob/release-0.27/Chart.yaml#L3-L4

File: charts/astronomer/values.yaml (hunk 1 above)
Comment (vishwas-astro) [reply charts/astronomer/values.yaml:27]: both 0.27.0 and release-0.27 exist in https://quay.io/repository/astronomer/ap-houston-api?tab=tags.

0.27.0 is explicitly this first version on the release branch and release-0.27 will be the tag published whenever the next houston build on release-0.27 branch happens.

**Pull request closed without merging**